desktop_version/src/DiscordNetwork.c

```diff
@@ -0,0 +1,165 @@
+#include "MakeAndPlay.h"
+
+#ifndef MAKEANDPLAY
+
+#include <stdint.h>
+#include <stdio.h>
+#include <assert.h>
+#include <SDL.h>
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <unistd.h>
+#include <string.h>
+#endif
+
+#include "Vlogging.h"
+
+// Library includes
+#include "discord_game_sdk.h"
+
+#if defined(_WIN32)
+#define DISCORD_LIBRARY "discord_game_sdk.dll"
+#elif defined(__APPLE__)
+#define DISCORD_LIBRARY "libdiscord_game_sdk.dylib"
+#elif defined(__linux__) || defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__HAIKU__) || defined(__DragonFly__)
+#define DISCORD_LIBRARY "libdiscord_game_sdk.so"
+#else
+#error DISCORD_LIBRARY: Unrecognized platform!
+#endif
+
+
+#define DISCORD_CRASH_LIMIT 10
+#define DISCORD_CLIENT_ID 1315544357532729447 // TO TERRY/FLIBIT: You can create your own Discord instance at the Discord Developer Portal. This ID belongs to me, so just be aware that if my account was to get hacked, VVVVVV RPC would too. Use your own!
```

Comment:
I feel like this comment gets added to the `#define` and ends up commenting out the semicolon where it's used? Might be wrong though.

It's good that this comment exists either way - the ID does need to be changed for production. I wonder if it'd be even better as a compile-time flag rather than hardcoded in...

Reply:
Well, I moved it just incase